src/include/tcp.h

```diff
@@ -38,6 +38,6 @@ int tcp_wcommit(struct tcp_chan *chan, int);
 int tcp_rskip(struct tcp_chan *chan,size_t);
 int tcp_chan_has_data(struct tcp_chan *chan);
 
-extern time_t pbs_tcp_timeout;
+extern "C" time_t pbs_tcp_timeout;
```

Comment:
This causes an error with C, needs to be something like:
```suggestion
#ifdef __cplusplus
extern "C" time_t pbs_tcp_timeout;
#else
extern time_t pbs_tcp_timeout;
%endif
```